1.0.1 doc changes

cs-abhishek-shukla opened the pull request:
1.0.1 Changes
- Fixed some links
- Added dependency on Code Snippet Connector under setup.md
- Added latest doc inputs from Rohit

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,103 +1,152 @@
+It is recommended that in `Production Mode` before you execute **Provide DPO and DPA Contact** playbooks, make the following changes
+1. Go to **Provide DPO and DPA Contact** playbook
+2. Open the *Get DPA Contact* step and remove `_demo` from `dpaEmail` variable as shown
+    ![Provide DPO and DPA Contact Changes](res/provide-dpo-and-dpa-contact.png)
```
Broken screenshot link
![image](https://user-images.githubusercontent.com/24773177/194795138-ca48f835-94e3-47f3-aea5-bb77d04d0179.png)

cs-abhinav-goel replied:
At multiple places the images have broken link.

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,103 +1,152 @@
+  ![Execution Mode Configuration](res/execution-mode-configuration.png)
 
+2. Modify the value of the `testMode` variable to `false`
+3. Change the value of the `testEmail` variable to a valid email address.
```
To move to production mode, user needs to give blank value for the variable instead of giving a valid value.

While in case of test mode user needs to give a valid email address so that all the email should go to the respective email address

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,103 +1,152 @@
-    ![notifyAffectedUsers](res/notify-affected-users.png)
+## Get Updated Report from DPO
+- If DPO rejects the data breach report, then this new task is created where the DPO can update the Data Breach Report and submit a new data
+- Open the task and click on the `Get Updated Breach Report` button
```
Need to the image when DPO sends the updated report.

cs-rohit-khune commented on docs/setup.md:
```diff
@@ -1,20 +1,65 @@
-For optimal performance of **GDPR Framework** solution pack, configure **Code Snippet** connector to extract and build address book of geo-specific DOP contacts.
-- The connector comes pre-installed and hence needs only configurations. To configure **Code Snippet** connector, refer to [Configuring Code Snippet](https://docs.fortinet.com/document/fortisoar/0.0.0/fortisoar-built-in-connectors/1/fortisoar-built-in-connectors#CodeSnippet).
\ No newline at end of file
+For optimal performance of **GDPR Framework** solution pack, configure **Code Snippet** connector to extract and build an address book of geo-specific DOP contacts.
```
>> For optimal performance of **GDPR Framework** solution pack, configure **Code Snippet** connector to extract and build an address book of geo-specific DOP contacts.

@cs-abhishek-shukla Please replace `DOP` with `DPA`[stands for Data Protection  Authority] in this sentence

cs-rohit-khune commented on docs/usage.md:
```diff
@@ -1,103 +1,152 @@
+  ![Execution Mode Configuration](res/execution-mode-configuration.png)
+
+2. Modify the value of the `testMode` variable to `true`
+3. Change the value of the `testEmail` variable to `null`.
```
For test mode, all possible combinations need to be added. Please refer below

           > - testMode = `true` and testEmail = `<Valid Email ID>`
           > - testMode = `true` and testEmail = `null`
           > - testMode = `false` and testEmail = `<Valid Email ID>`

cs-abhishek-shukla committed:
1.0.1 Changes
- Fixed some links
- Added dependency on Code Snippet Connector under setup.md
- Added latest doc inputs from Rohit and Abhinav
